etc/picongpu/summit-ornl/gpu_picongpu.profile.example

```diff
@@ -33,20 +33,20 @@ module load boost/1.66.0
 module load hdf5/1.10.3
 module load adios/1.13.1-py2 c-blosc zfp sz lz4
 
-# download libSplash and compile it yourself from
+# optionally download libSplash and compile it yourself from
 #   https://github.com/ComputationalRadiationPhysics/libSplash/
-export Splash_DIR=$PROJWORK/$proj/huebl/sw/splash-1.7.0
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$Splash_DIR/lib
+# export Splash_DIR=<your libSplash install directory>
```

Comment:
These days, e.g. in the `cori-nersc` profile, I usually recommend some path such as
```bash
export Splash_ROOT=${HOME}/sw/libSplash-1.7.0
```

Another example can be seen in `pizdaint-cscs` which is close to that.

Just trying to keep it similar so we are not introducing yet another naming scheme.

Reply:
Thanks, I will change to `_ROOT`. Before just kept the naming scheme in that file.